Makefile

```diff
@@ -182,8 +184,10 @@ storage:
 kafka:
 	@echo Creating namespace $(KAFKA_NAMESPACE)
 	@kubectl create namespace $(KAFKA_NAMESPACE) 2>&1 | grep -v "already exists" || true
-	@sed 's/namespace: .*/namespace: kafka/' ./test/kafka-operator.yml | kubectl -n $(KAFKA_NAMESPACE) apply -f -  2>&1 | grep -v "already exists" || true
-	@kubectl apply -f ./test/kafka.yml -n $(KAFKA_NAMESPACE) 2>&1 | grep -v "already exists" || true
+	@curl --location ${KAFKA_YAML} --output ./test/kafka-operator.yml
+	@curl --location ${KAFKA_EXAMPLE} --output ./test/kafka-example.yaml
```

Comment:
I see that those files were added to this PR. Is that intentional? If not, this should be added to the gitignore, or piped directly to kubectl, like:

```console
curl ${KAFKA_YAML} | sed ... | kubectl apply -n ... -f -
```

Perhaps this would be better suited for a new script, `./.ci/setup-kafka.sh`?

Reply:
+1 for a direct pipe